.github/workflows/ci-matrix-build.yml

```diff
@@ -0,0 +1,69 @@
+#This workflow runs matrix build with both versions of setup MATLAB action hourly
+
+name: MATLAB matrix build
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [main, setup-matlab-matrix-build]
+  pull_request:
+    branches: [main]
```

Comment:
Not sure if we need to run this workflow on pushes or pull requests. We might be able to remove  this and just keep the schedule.